judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/interrelationship/InterrelationshipController.java

```diff
@@ -0,0 +1,54 @@
+package dev.hltech.dredd.interfaces.rest.interrelationship;
+
+import dev.hltech.dredd.domain.contracts.ServiceContracts;
+import dev.hltech.dredd.domain.contracts.ServiceContractsRepository;
+import dev.hltech.dredd.domain.environment.EnvironmentRepository;
+import dev.hltech.dredd.interfaces.rest.contracts.ContractsMapper;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.HashMap;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@RestController
+public class InterrelationshipController {
+
+    private final EnvironmentRepository environmentRepository;
+    private final ServiceContractsRepository serviceContractsRepository;
+    private final ContractsMapper contractsMapper;
+
+    @Autowired
```

Comment:
use lombok?

Reply:
It was not used throughout the project in this case, we should introduce it globally